pkg/snowflake/materialization.go

```diff
@@ -154,3 +155,32 @@ func buildTimeIntervalQuery(asset *pipeline.Asset, query string) (string, error)
 
 	return strings.Join(queries, ";\n") + ";", nil
 }
+
+func BuildDDLQuery(asset *pipeline.Asset, query string) (string, error) {
```

Comment:
uppercase functions are exposed to other packages in Go, in this case I don't think you want to expose this.
```suggestion
func buildDDLQuery(asset *pipeline.Asset, query string) (string, error) {
```

Reply:
Thanks for the tip! Changed now.